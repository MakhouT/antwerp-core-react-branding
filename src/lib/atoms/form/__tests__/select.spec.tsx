import * as React from "react";

import * as A from "../../..";
import { renderAndMatchSnapshot } from "../../../../__tests__/__fixtures__/testUtils";

describe("Select", () => {
    it("Renders default", () => {
        // Arrange
        const element = <A.Select
            name="select"
            options={
                [
                    { label: "a", value: "a" },
                    { label: "b", value: "b" },
                    { label: "c", value: "c" },
                ]
            }

        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });
});
