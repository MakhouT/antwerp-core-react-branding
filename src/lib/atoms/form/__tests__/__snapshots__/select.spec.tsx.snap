// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select Renders default 1`] = `
<div
  className="a-input has-icon-right"
>
  <div
    className="a-input__wrapper"
  >
    <select
      id="-undefined-select"
      name="select"
      onChange={[Function]}
      value=""
    >
      <option
        value=""
      />
      <option
        disabled={undefined}
        id="-checkbox-select-0"
        value="a"
      >
        a
      </option>
      <option
        disabled={undefined}
        id="-checkbox-select-1"
        value="b"
      >
        b
      </option>
      <option
        disabled={undefined}
        id="-checkbox-select-2"
        value="c"
      >
        c
      </option>
    </select>
    <span
      className="fa fa-angle-down"
      onClick={[Function]}
      style={undefined}
    />
  </div>
  <small
    className=""
  >
     
  </small>
</div>
`;
